scala2.13/pom.xml

```diff
@@ -810,6 +810,7 @@
             351
         </noSnapshot.buildvers>
         <snapshot.buildvers>
+            400
```

Comment:
Need to regenerate scala2.13 pom

Reply:
Done. I was curious why we need to commit the 2.13 pom when it can be generated when needed.

Reply:
In other words, isn't this a lot similar to generating shims for different versions of Spark?

Reply:
It's committed for convenience.  Developers can directly point their IDE to the scala2.13 pom or directly build after pulling source.  If it required manual generation, switching branches in your local repo would be fraught with problems if you forget to re-generate the scala2.13 pom after moving to a new commit.  Something that would be very easy to do.